CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# 2.0.0
+
+## Dependency Update
+
+The following packages are now specified as peer dependencies:
+
+* react
+* react-dom
+* flux
+
+The following packages have been moved from `devDependencies` to `dependencies`:
+
+* `i18n-js`
+* `immutable`
+* `react-router`
+* `react-transition-group`
```

Comment:
It would be useful to have guides on how to upgrade to this version (eg. what do they need to change in their `package.json` file?)